src/schematools/contrib/django/management/commands/create_tables.py

```diff
@@ -92,14 +93,20 @@ def create_tables(
                 command.stderr.write(f"  Skipping non-managed model: {model._meta.db_table}")
                 continue
 
-            try:
-                command.stdout.write(f"* Creating table {model._meta.db_table}")
-                with transaction.atomic():
-                    schema_editor.create_model(model)
-            except (DatabaseError, ValueError) as e:
-                command.stderr.write(f"  Tables not created: {e}")
-                if not re.search(r'relation "[^"]+" already exists', str(e)):
-                    errors += 1
+            if dry_run:
+                command.stdout.write(
+                    f"* Would create table {model._meta.db_table} if it doesn't exist"
+                )
+            else:
+                try:
+                    with transaction.atomic():
+                        schema_editor.create_model(model)
+                except (DatabaseError, ValueError) as e:
+                    command.stderr.write(f"  Cannot create table {model._meta.db_table}: {e}")
+                    if not re.search(r'relation "[^"]+" already exists', str(e)):
+                        errors += 1
+                else:
+                    command.stdout.write(f"* Creating table {model._meta.db_table}")
```

Comment:
```suggestion
                    command.stdout.write(f"* Created table {model._meta.db_table}")
```

Minor nitpick, as table creation will already be done when reaching here